facebook/__init__.py

```diff
@@ -122,6 +122,29 @@ def get_objects(self, ids, **args):
         args["ids"] = ",".join(ids)
         return self.request(self.version + "/", args)
 
+    def search_objects_by(self, type, **args):
+        """Fetches all objects of a given type from the graph.
+
+        Valid types are: event, group, page, place, placetopic, and user
```

Comment:
This documentation should go in the [SDK's API reference](https://github.com/mobolic/facebook-sdk/blob/master/docs/api.rst) as opposed to its docstring, just like the SDK's other methods.